include/hax_interface.h

```diff
@@ -143,6 +143,11 @@ struct hax_tunnel {
         struct {
             paddr_t dummy;
         } state;
+        struct {
+            uint64_t rip;
+            uint64_t dr6;
+            uint64_t dr7;
+        } debug;
```

Comment:
> Are there any debugging features we could support in the future that might require passing additional data to QEMU?

I don't think so. On the SDM I've seen other nice features that could benefit debuggers (e.g. tracing), but they can have their own *ioctl* since they are too wildly different from breakpoints/stepping and considerably more complex.

So I don't think we will ever need more than these 24 bytes.